ocaml/src/asd_config_test.ml

```diff
@@ -0,0 +1,55 @@
+(*
+Copyright (C) 2016 iNuron NV
+
+This file is part of Open vStorage Open Source Edition (OSE), as available from
+
+
+    http://www.openvstorage.org and
+    http://www.openvstorage.com.
+
+This file is free software; you can redistribute it and/or modify it
+under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+as published by the Free Software Foundation, in version 3 as it comes
+in the <LICENSE.txt> file of the Open vStorage OSE distribution.
+
+Open vStorage is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY of any kind.
+*)
+
+open OUnit
+open Asd_config
+let test_basic_config () =
```

Comment:
No, this test was supposed to see how the jsonification of constructors with parameters would go.
Nonetheless, it's a good idea to put that file under test. Will do so.